.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseTimePeriodParser.cs

```diff
@@ -204,10 +204,13 @@ private DateTimeResolutionResult MergeTwoTimePoints(string text, DateObject refe
 
                     foreach (var num in numErs)
                     {
+                        int midStrBegin = 0, midStrEnd = 0;
+                        // ending number
                         if (num.Start > ers[0].Start + ers[0].Length)
                         {
-                            // check if it is a ending number
-                            var endingNumber = false;
+                            midStrBegin = ers[0].Start + ers[0].Length ?? 0;
+                            midStrEnd = num.Start - midStrBegin ?? 0;
+                            /*var endingNumber = false;
```

Comment:
Please don't include commented out code in PRs to /master.
Can't this just be deleted?